README.md

```diff
@@ -78,7 +84,7 @@ then copies the results to a clean image.
 
 ``` shell
 # build the custom notconf-test image
-❯ docker build -f Dockerfile.yang -t notconf-test --build-arg COMPOSE_PATH=test .
+❯ docker build -f Dockerfile.yang -t notconf-dzs --build-arg COMPOSE_PATH=dzs/yang-modules .
```

Comment:
Right, "dzs" instead of "test" it is an oversight. 
The folder name `yang-modules` however is necessary so I corrected like this:

`❯ docker build -f Dockerfile.yang -t notconf-test --build-arg COMPOSE_PATH=test/yang-modules .`